dom.bs

```diff
@@ -2389,12 +2389,10 @@ of a <var>node</var> into a <var>parent</var> before a <var>child</var>, run the
 </ol>
 
 <p><a lt="Other applicable specifications">Specifications</a> may define
-<dfn export id=concept-node-insert-ext>insertion steps</dfn> for all or some <a for=/>nodes</a>. The
-algorithm is passed <var ignore>insertedNode</var>, as indicated in the <a for=/>insert</a>
-algorithm below.
-<!-- See https://github.com/whatwg/dom/issues/34#issuecomment-125571750 for why we might need to
-     adjust this further based on the requirements of the script element. There might be other ways
-     to define that though as Olli suggests, so leaving that out for now. -->
+<dfn export id=concept-node-insert-ext>insertion steps</dfn> or
+<dfn export id=concept-node-children-added>children added steps</dfn> for all or some
+<a for=/>nodes</a>. The two algorithms are passed <var ignore>insertedNode</var> and
+<var ignore>deferredStepsQueue</var>, as indicated in the <a for=/>insert</a> algorithm below.
```

Comment:
Should we mention child text content change steps here?

Reply:
I don't know, they are defined elsewhere with `Text` and they don't take `deferredStepsQueue`.

Reply:
I see, I think it makes sense to group them, but that'd be a follow-up indeed.